wordpress-persistent-disks/mysql-service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysql
+  labels:
+    app: mysql
+spec:
+  type: ClusterIP
```

Comment:
We usually just let it default to this? Makes sense and is cleaner to me but maybe being explicit is better, idk.